BFWControls/Modules/Table/Controller/StaticTableViewController.swift

```diff
@@ -85,6 +88,29 @@ class StaticTableViewController: UITableViewController {
         return IndexPath(row: (indexPath as NSIndexPath).row + numberOfExcludedRows(before: indexPath),
                          section: (indexPath as NSIndexPath).section)
     }
+	
+	fileprivate func refreshTableView() {
```

Comment:
We already have updateCellHeights function from UITableViewController extension in MOA